data/analytics/events.yml

```diff
@@ -384,7 +384,7 @@
         - name: event_name
           value: navigation
         - name: external
-          value: true
+          value: string (true)
```

Comment:
I think this is a bit confusing, although I get that you're trying to clarify that the `true` value is a string. Maybe instead if we had it display as

```
  external: "true"
```

...although that might require some code changes to make the quotes render.

There might be a separate conversation to be had about how much information we want to display in the event about the attribute (for example we could put all the value types after the values e.g. `external: "true" (string)` but I'd be concerned about overloading this view and reducing the value of the attribute page itself.

Reply:
No worries, I can just do `external: '"true"'` instead :+1: